components/charts/chartsjs/BarChart.js

```diff
@@ -72,14 +72,19 @@ const DeathsByDataType = props => {
       yAxes: [
         {
           ticks: {
-            suggestedMax: scaleMax,
+            suggestedMax: scaleMax + 100,
```

Comment:
What do you think about making `suggestedMax` some multiple of `scaleMax` instead of adding a fixed value here? This `suggestedMax` looks good when all the filters are enabled and the `scaleMax` is 780, but when I check just the "not given" race, `scaleMax` is 7, and all the bars are very short.

```suggestion
            suggestedMax: scaleMax * 1.1,
```

Reply:
(FWIW, I didn't check to see if this number needs to be an integer. In that case, I think we'd need to `Math.ceil(scaleMax * 1.1)`.)

Reply:
Thanks @nholden , I like that idea!